docs/en/sql-reference/functions/ip-address-functions.md

```diff
@@ -243,4 +243,37 @@ SELECT
 └───────────────────────────────────┴──────────────────────────────────┘
 ```
 
+
+## isIPv4String
+
+Determines if the input string is an IPv4 address or not. Returns `1` if true `0` otherwise.
+
+``` sql
+SELECT isIPv4String('127.0.0.1')
+
+Query id: 031fdb77-fa98-456e-9323-c68c7788fdde
```

Comment:
Lets' remove `Query id` and `1 rows in set. Elapsed:` from examples. Also we can split query and response in separate code block`sql` and `text` as in examples above.

Reply:
Sure, fixed..